tests/common/fixtures/__init__.py

```diff
@@ -1,3 +0,0 @@
-from tests.common.fixtures.elasticsearch import es_client, init_elasticsearch
-
-__all__ = ("es_client", "init_elasticsearch")
```

Comment:
This is just getting in the way. We directly import from other things in this directory